launcher/src/background.js

```diff
@@ -33,10 +34,24 @@ const tekuGasLimitConfig = new TekuGasLimitConfig(nodeConnection);
 const sshService = new SSHService();
 const { globalShortcut } = require("electron");
 const log = require("electron-log");
+const logFileBackup = new LogFileBackup();
 const stereumUpdater = new StereumUpdater(log, createWindow, isDevelopment);
 stereumUpdater.initUpdater();
 log.transports.console.level = process.env.LOG_LEVEL || "info";
 log.transports.file.level = "debug";
+log.transports.file.archiveLogFn = (file) => {
+  file = file.toString();
+  const info = path.parse(file);
+  let backupPath = info.dir + "/backups/";
+  if (!existsSync(backupPath)) {
+    mkdirSync(backupPath);
+  }
+
```

Comment:
the main.log is not renamed or moved, this will not store new logs after 1 mb if i'm not mistaken.